modules/packages/yum

```diff
@@ -103,8 +103,13 @@ def list_installed():
 
 
 def list_updates(online):
-    # Ignore everything.
-    sys.stdin.readlines()
+    for line in sys.stdin:
+        line = line.strip()
+        if line.startswith("options="):
+            option = line[len("options="):]
+            if option.startswith("enablerepo=") or option.startswith("disablerepo="):
+                global yum_options
+                yum_options += ["--" + option]
```

Comment:
There's no need to use ``global`` when modifying an existing object. Also, ``yum_options.append()`` would be less cryptic here, I think. And the same of course applies below too.